webpack_loader/loader.py

```diff
@@ -85,6 +85,8 @@ def get_bundle(self, bundle_name):
         elif assets.get('status') == 'error':
             if 'file' not in assets:
                 assets['file'] = ''
+            if 'error' not in assets:
+                assets['error'] = 'error'
```

Comment:
I could totally change it to `Unknown Error` (cause `in` is added in the string further down)